src/explore-education-statistics-admin/src/components/NavigableSections.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactNode } from 'react';
+import PreviousNextLinks from '@admin/components/PreviousNextLinks';
+import Link from '@admin/components/Link';
+
+export interface Section<SectionType extends {}> {
+  section: SectionType;
+  label: string;
+  linkTo: string;
+}
+
+interface Props<SectionType> {
+  navigationHeadingText: string;
```

Comment:
It doesn't look like we're actually using either of the `navigationHeadingText` and `navigationHeadingSubtitle` props in the component?

Reply:
ahh good spot, one snuck through!  It previously was used but in the end it felt that the page heading belonged in the page and not this component, which'd make this component more reusable.  Will remove the above.

Reply:
Done, although the whole component is now gone too, so bah!